config/config_helpers.go

```diff
@@ -318,22 +310,43 @@ var listQuotedParamRegex = regexp.MustCompile(`^"([^"]*?)"\s*,\s*(.*)$`)
 // foo("a", "b") -> return "a", "b", 2, nil
 //
 func parseOptionalQuotedParam(parameters string) (string, string, int, error) {
+	parsed, err := splitParamList(parameters)
+
+	if err != nil {
+		return "", "", 0, err
+	}
+
+	if len(parsed) == 0 {
+		return "", "", 0, nil
+	} else if len(parsed) == 1 {
+		return parsed[0], "", 1, nil
+	} else if len(parsed) == 2 {
+		return parsed[0], parsed[1], 2, nil
+	}
+	return "", "", 0, errors.WithStackTrace(InvalidStringParams(parameters))
+}
+
+func splitParamList(parameters string) ([]string, error) {
 	trimmedParameters := strings.TrimSpace(parameters)
 	if trimmedParameters == "" {
-		return "", "", 0, nil
+		return []string{}, nil
 	}
 
-	matches := oneQuotedParamRegex.FindStringSubmatch(trimmedParameters)
-	if len(matches) == 2 {
-		return matches[1], "", 1, nil
+	if !INTERPOLATION_PARAMETERS_REGEX.MatchString(trimmedParameters) {
```

Comment:
Here, as mentioned above, I re-validate that the input string is a valid parameter-list.  Normally, this wouldn't happen but there are a few tests that call this function directly.